appendices/editor-initial-response-template.md

```diff
@@ -5,12 +5,12 @@
 
 ## Editor comments
 :wave: Hi @reviewer-one and @reviewer-two! Thank you for volunteering to review
-for PyOpenSci! <Add any additional banter here that you wish..>
+for pyOpenSci! <Add any additional banter here that you wish..>
```

Comment:
@Batalex good catch!! gosh i thought i had caught all of the broken links but alas, i missed more!! THANK YOU!